.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2021 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    -   id: check-merge-conflict
+-   repo: https://github.com/psf/black
+    rev: 21.7b0
+    hooks:
+    -   id: black
+-   repo: https://github.com/PyCQA/isort
+    rev: 5.9.3
+    hooks:
+    -   id: isort
+        args: ["--profile", "black"]
```

Comment:
Isn't this setting already set in `pyproject.toml`?

Reply:
It is, but when I run it locally to automatically fix files before commit it doesn't use pyproject.toml

Reply:
Sounds strange, it seems to be working for other repositories.

Reply:
Can you provide an example of it not taking `pyproject.toml` into account?

Reply:
I can't reproduce this issue, so `args: ["--profile", "black"]` is not necessary.